HLTriggerOffline/Btag/interface/HLTBTagHarvestingAnalyzer.h

```diff
@@ -48,6 +49,8 @@ class HLTBTagHarvestingAnalyzer : public DQMEDHarvester {
 			// Histogram handler
 			std::map<std::string, MonitorElement *> H1_;
 
+			std::vector<std::string> modules_;
```

Comment:
hi @natalia-korneeva - sorry I missed your answer - I see your point now - so the needed change to make everything work is to make a small map:
enum HCALSpecials = { HEP17, HEP18, HEM17};
std::map<HCALSpecials,std::string> hcalSpecialNames = { HEP17: "HEP17", HEP18: "HEP18", HEM17: "HEM17" };

and then use the enums to index and look up the histograms, and the map to get from enum to string that you need for constructing the histograms.

Does that make sense?